build/opusfile-0.7/win32/VS2015/opusfile.vcxproj

```diff
@@ -108,7 +108,7 @@
       <PreprocessorDefinitions>WIN32;_DEBUG;_CRT_SECURE_NO_WARNINGS;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
       <ProgramDataBaseFileName>$(OutDir)$(TargetName).pdb</ProgramDataBaseFileName>
-      <AdditionalIncludeDirectories>../../include;../../../../include/opus</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>../../include;../../../opus-1.1.2/include</AdditionalIncludeDirectories>
```

Comment:
Could you please leave this as referring to %INCLUDEDIR%? 

Ideally, every project in build/ should be entirely independent, and get their dependencies directly from %LIBDIR% and %INCLUDEDIR%.